q2_longitudinal/plugin_setup.py

```diff
@@ -31,7 +31,13 @@
         'This QIIME 2 plugin supports methods for analysis of time series '
         'analysis, involving either paired sample comparisons or longitudinal '
         'study designs.'),
-    short_description='Plugin for paired sample and time series analyses.'
+    short_description='Plugin for paired sample and time series analyses.',
+    citation_text=("q2-longitudinal: a QIIME 2 plugin for longitudinal and "
+                   "paired-sample analyses of microbiome data. Nicholas "
+                   "Bokulich, Yilong Zhang, Matthew Dillon, Jai Ram Rideout, "
+                   "Evan Bolyen, Huilin Li, Paul Albert, James Gregory "
```

Comment:
Could you change _James_ to _J_?

Reply:
👍 sorry — was just copy/pasting the author list from biorxiv.

fixed.

Reply:
Thanks!